app/public/static/js/components/keypad.js

```diff
@@ -18,7 +18,8 @@ AFRAME.registerComponent("keypad", {
 
     el.setAttribute(
       "super-keyboard",
-      "imagePath:/static/img/; multipleInputs:true; model: numpad; align: center; maxLength: 4;"
+      "imagePath:/static/img/; multipleInputs:true; align: center; maxLength: 4; label: Enter Password; labelColor: black; model:" +
```

Comment:
I think you have an extra label and labelColor here :)

Reply:
Thanks for catching that, forgot to remove it when I merged from master with the latest changes.